src/runtime/initialize-component.ts

```diff
@@ -32,7 +32,7 @@ export const initializeComponent = async (
     // Let the runtime know that the component has been initialized
     hostRef.$flags$ |= HOST_FLAGS.hasInitializedComponent;
 
-    if (BUILD.lazyLoad || BUILD.hydrateClientSide) {
+    if (BUILD.lazyLoad) {
```

Comment:
Can you explain the reasoning for this change? For me it seems that keeping `|| BUILD.hydrateClientSide` results in the same result 🤔

Reply:
Ok, I see it prevents Stencil from lazily loading a component file.